### DISCUSSION TITLE
<podcast:person gender="m/f/d/*">

### DISCUSSION THREAD

**Discussion by saerdnaer** [Podcastindex-org/podcast-namespace#495] [Enhancement Proposal]
Should we add a gender attribute to the person tag?

**Comment by keunes** [comment]
Why would we need that?

**Comment by saerdnaer** [comment]
For example if you want to search for episodes/podcasts with at least one female contributor

**Comment by keunes** [comment]
Hmm, ok. I don't see the appeal of it right now; I would mainly use person info to search for guests or hosts that I know are interesting. But maybe there's value in the use-case you describe.

**Comment by theDanielJLewis** [comment]
While this would be interesting for data analysis, such would also lack enough adoption to be worth much.

And I don't see much need for searching a podcast based solely on whether the hosts are male or female.

If we did support gender, then I foresee all kinds of additional requests, like sexual preference, age, race, etc.

I would also worry that even a gender tag could lead to discrimination or worse.

**Comment by PofMagicfingers** [comment]
There is many langages, including French, where gender neutral is sometimes
tricky. Having the gender/pronom of the guest and host of a podcast could
be useful for public profiles and other ui elements. "See her/his profile",
etc.

While this would be interesting for data analysis, such would also lack

**Comment by keunes** [comment]
> There is many langages, including French, where gender neutral is sometimes tricky. Having the gender/pronom of the guest and host of a podcast could be useful for public profiles and other ui elements. "See her/his profile"

Seems this example, at least for French, isn't really working as an argument here: the possessive adjective in French takes the gender of the possessed object, not of the owner: « son profil » is both 'his' and 'her' profile. In English 'their' is quite well accepted as a gender-neutral form, and workarounds are used in many other languages (often 'his/her').

**Comment by daveajones** [comment]
I'll keep this open for comment, but this one seems a little creepy.  Just my two cents.
